eng/Generation.psm1

```diff
@@ -50,6 +50,59 @@ function AutoRest-Reset()
     Invoke "$script:autoRestBinary --reset"
 }
 
+function Invoke-Cadl($baseOutput, $projectName, $sharedSource="", $fast="", $debug="")
```

Comment:
Lack of the argument of `arguments`

Reply:
There is no function argument 'arguments'